src/AmyrisBio/primercore2.fs

```diff
@@ -461,7 +561,11 @@ module primercore =
                         if debug then printfn "designLeft, stopping at nextBase-1=%d thisTemp=%f" (nextBase-1) (thisTemp/1.0<C>)
                         Some(cutToGC debug lastTemp p s 0 (nextBase-1) gcCount offset)
 
-        designLeftInternal sInitUpper nextBase gcCount (0.0<C>)
+        match designLeftInternal sInitUpper nextBase gcCount (0.0<C>) with
```

Comment:
This could be cleanly refactored using Option.map:
```fsharp
designLeftInternal sInitUpper nextBase gcCount (0.0<C>)
|> Option.map (fun prePDCheck ->
    disruptPrimerDimers debug prePDCheck.temp p sInitUpper 0 (prePDCheck.oligo.Length-1) offset)
```

Reply:
Nice.  I learn something every time you review my code :)